service/service.hpp

```diff
@@ -241,13 +262,21 @@ namespace raven
         std::cout << "cfg.id: " << cfg.id << std::endl;
         std::cout << "cfg.setting_name: " << cfg.setting_name << std::endl;
         std::cout << "cfg.alias_name: " << cfg.alias_name << std::endl;
+
+        json::json response_json_data;
+        response_json_data[request_state_keyword] = convert_request_state.at(request_state::success);
+        send_answer(std::move(response_json_data), sock);
```

Comment:
This block seems to be repeated many times. Would it be possible to refactor it ?